### PR TITLE
Fix 1 byte reads (extremely slow)

### PR DESCRIPTION
**Pull request by vansante**

### CODE REVIEW

File: exif/exif.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ const (
 	interopPointer = 0xA005
 )
 
+const APP_SEC_READ_BUFFER = 8 * 1024 // 8 KiB
```
Comment (xor-gate) [inline exif/exif.go]: Why don't we hardcode 32K just as in the stdlib copybuffer? See https://golang.org/src/io/io.go#L387. IMHO this public constant should not be visible (or hardcoded)

**Comment by vansante** [comment]
Makes sense, changed it.

**Comment by xor-gate** [comment]
Thanks, I'm not 100% sure why 32K pages are chosen but probably they have a very good reason for that.

**Comment by xor-gate** [comment]
Merged, thank you for your contribution!

**Pull request merged**